pom.xml

```diff
@@ -378,6 +378,7 @@
                         <systemPropertyVariables>
                            <!-- See test org.hibernate.ogm.test.utils.EnvironmentTest -->
                            <hibernate.service.allow_crawling>false</hibernate.service.allow_crawling>
+                           <user.timezone>UTC</user.timezone>
```

Comment:
What kind of error?
The time  zone should be already considered when converting the types.
For example, `BuiltInTypeTest` set the  time zone before running the tests so it should not fail. I'm asking because I would like to be sure about the cause of the error and understand if it is something in the test or in the dialect.